spec/fixtures/factories.rb

```diff
@@ -190,4 +190,18 @@
       }
     end
   end
+
+  factory :maib_report, parent: :document do
+    base_path "/maib-reports/example-maib-report"
+    document_type "maib_report"
+
+    transient do
+      default_metadata {
+        {
+          "date_of_occurrence" => "2015-10-10",
+          "document_type" => "maib_report",
```

Comment:
I think the document type in metadata was introduced first, and at some point later, document type was added to the top level as well. I reckon that we could simplify the schema to get rid of the `document_type` in the `metadata` block /cc @tuzz